ui/raidboss/emulator/data/NetworkLogConverter.worker.ts

```diff
@@ -25,6 +25,14 @@ ctx.addEventListener('message', (msg) => {
   localLogHandler.on(
     'fight',
     (day: string, zoneId: string, zoneName: string, lines: LineEvent[]) => {
+      // Index the lines for this encounter
+      for (let i = 0; i < lines.length; ++i) {
+        const line = lines[i];
+        if (!line)
+          throw new UnreachableCode();
+        line.index = i;
```

Comment:
Maybe grep is failing me, but where does `index` get used?

Reply:
It's going to be used in an upcoming PR for the breaking up of 3360, and it's useful for debugging purposes in general (which is why it was included originally).